server/src/test/java/com/cloud/resourcelimit/ResourceLimitManagerImplTest.java

```diff
@@ -472,6 +508,51 @@ public void testFindCorrectResourceLimitForDomain() {
         Assert.assertEquals(defaultDomainCpuMax, result);
     }
 
+    @Test
```

Comment:
again, can this test case be split?

Reply:
alrighty, I was able to do the changes as you described. Please let me know if there is anything else I can do.

Reply:
ok, this one looks good, not sure if the build failures are related.